keyboards/s_ol/0xc_pad/keymaps/default/readme.md

```diff
@@ -0,0 +1,5 @@
+![0xC.pad Layout Image](https://i.imgur.com/RTC1LgB.png)
```

Comment:
Tiny note: the image here doubles the Page Up and Volume Up keycodes, instead of having Page Up, Page Down, Volume Up and Volume Down.

Reply:
Thanks, fixed in last push :)